kong/plugins/ddtrace/handler.lua

```diff
@@ -32,8 +32,10 @@ end
 -- This timer runs in the background to flush traces for all instances of the plugin.
 -- Because of the way timers work in lua, this can only be initialized when there's an
 -- active request. This gets initialized on the first request this plugin handles.
-local agent_writer_timer
+local agent_writer_timer -- luacheck: ignore 231
```

Comment:
I spent undue time looking into this luacheck warning and the inner workings of `ngx.timer.every`. Let's discuss in our meeting. TL;DR what you have here is fine.